Front-end/src/Profile.jsx

```diff
@@ -98,19 +100,83 @@ const Profile = (props) => {
       })
     );
   }
+  function Followers() {
+    fetch(`http://localhost:3000/followers/${profile.id}`, {
+      method: "GET",
+      headers: {
+        "Content-Type": "Application/json",
+      },
+    }).then((data) =>
+      data.json().then((data) => {
+        setFollowers(data);
+      })
+    );
+  }
+  function SetFollow() {
+
+    fetch(`http://localhost:3000/followUser/${user.id}/${profile.id}`, {
+      method: "GET",
+      headers: {
+        "Content-Type": "Application/json",
+      },
+    }).then((data) =>
+      data.json().then((data) => {
+        if (data === 0) {
+          setStatus("Follow");
+        } else {
+          setStatus("Unfollow");
+        }
+      })
+    );
+  }
+  function handleFollow(){
+    if (status === "Follow") {
+      console.log("TEST", profile.id)
+      fetch(`http://localhost:3000/follow/${profile.id}`, {
+      method: "POST",
+      headers: {
+        "Content-Type": "Application/json",
+      },
+      body: JSON.stringify({ followId: user.id})
+    }).then((data) =>
+      data.json().then((data) => {
+        setStatus("Unfollow");
+      })
+    );
+    } else {
+      fetch(`http://localhost:3000/follow/${profile.id}`, {
+      method: "DELETE",
+      headers: {
+        "Content-Type": "Application/json",
+      },
+      body: JSON.stringify({ followId: user.id})
+    }).then((data) =>
+      data.json().then((data) => {
+        setStatus("Follow");
+      })
+    );
+    }
+  }
 
+useEffect(() => {
+  Auth();
+}, [])
   useEffect(() => {
-    Auth();
-    if (user && func === false) {
+    if(user) {
```

Comment:
space `if (user) {`

Reply:
This code section is outdated. In the main branch it is corrected.